CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ separate changelogs for each crate were used. If you need to refer to these old
 - `libcnb-package`: buildpack target directory now contains the target triple. Users that implicitly rely on the output directory need to adapt. The output of `cargo libcnb package` will refer to the new locations. ([#580](https://github.com/heroku/libcnb.rs/pull/580))
 - `libherokubuildpack`: Switch the `flate2` decompression backend from `miniz_oxide` to `zlib`. ([#593](https://github.com/heroku/libcnb.rs/pull/593))
 - Bump minimum external dependency versions. ([#587](https://github.com/heroku/libcnb.rs/pull/587))
+- `libcnb-cargo`: Default location for packaged buildpacks moved from Cargo's `target` directory to `libcnb-packaged` in the Cargo workspace root. This simplifies the path and stops modification of the `target` directory which previously might have caching implications when other tools didn't expect non-Cargo output in that directory. Users that implicitly rely on the output directory need to adapt. The output of `cargo libcnb package` will refer to the new locations. ([#583](https://github.com/heroku/libcnb.rs/pull/583))
```

Comment:
There's a `libcnb-package` changelog entry a couple of lines above this one that perhaps needs to be combined with this new entry? (Since the path has been changed twice)

Reply:
This is a weird one. The target triple in the output path is controlled by `libcnb-package`. The new output directory root however is a `libcnb-cargo` change only. `libcnb-package` always got an explicit root path and created the distribution under that one.

So technically, these are two separate changes. Is that easy to understand for users `cargo libcnb package`? 💯  not. I'm not really sure which solution I like the best here. In the majority of cases, `libcnb-package` is only used as an internal dependency so that changes to it could be considered changes of `cargo-libcnb`.

We have a similar situation with `libcnb-data` where it's not always clear where the changelog should go.